source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -1115,6 +1125,22 @@ clear the pool again. This situation is possible if the pool is cleared by the
 background thread after it encounters an error establishing a connection, but
 the ServerDescription for the endpoint was not updated accordingly yet.
 
+Why do we need force running prune maintenance call in Clear logic?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If a SDAM logic has noticed a network error, we assume that all connections including
```

Comment:
Should "network timeout" be used instead of a general "network error"?

Reply:
true, done